content/warnings/special-props.md

```diff
@@ -4,6 +4,6 @@ layout: single
 permalink: warnings/special-props.html
 ---
 
-Most props on a JSX element are passed on to the component, however, there are two special props (`ref` and `key`) which are used by React, and are thus not forwarded to the component.
+Większość właściwości (ang. *props*) elementów jest przekazywanych do komponentu. Istnieją jednak dwie specjalne właściwości (`ref` oraz `key`), których używa na wyłączność React, a co za tym idzie, nie trafiają do komponentu.
 
-For instance, attempting to access `this.props.key` from a component (i.e., the render function or [propTypes](/docs/typechecking-with-proptypes.html#proptypes)) is not defined. If you need to access the same value within the child component, you should pass it as a different prop (ex: `<ListItemWrapper key={result.id} id={result.id} />`). While this may seem redundant, it's important to separate app logic from reconciling hints.
+Na przykład, próba odczytania wartości `this.props.key` w komponencie (w funkcji renderującej lub [propTypes](/docs/typechecking-with-proptypes.html#proptypes)) zakończy się niepowodzeniem. Jeśli potrzebujesz odczytać taką wartość w komponencie potomnym, przekaż ją ponownie pod inną nazwą (np. `<ListItemWrapper key={result.id} id={result.id} />`). Nawet jeśli wydaje się to nadmiarowe, ważne jest, aby oddzielać logikę aplikacji od wskazówek rekoncyliacyjnych (ang. *reconciling hints*).
```

Comment:
"Nadmiarowe" trochę dziwnie mi tutaj brzmi. Może lepiej:
> Nawet jeśli wydaje się to zbyteczne

Reply:
W oryginale było "redundant", co zwykle tłumaczy się w ten sposób. Niektórzy piszą "redundantne", ale brzmi to dla mnie dziwacznie.
Ale "zbyteczne" też ok :-)